rest-model/src/test/java/org/jboss/pnc/restmodel/serialization/BuildExecutionConfigurationTest.java

```diff
@@ -47,6 +47,7 @@ public void serializeAndDeserializeBuildResult() throws IOException, BuildDriver
                 "configuration name",
                 "https://pathToRepo.git",
                 "1111111",
```

Comment:
Shouldn't the SCM revision have sha format?

Reply:
Yes, it should. I'll fix it.